ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/messages/StatusMessage.java

```diff
@@ -216,4 +216,16 @@ public static EthStatus readFrom(final RLPInput in) {
           protocolVersion, networkId, totalDifficulty, bestHash, genesisHash, forkId);
     }
   }
+
+  @Override
+  public String toString() {
+    final StringBuilder stringBuilder = new StringBuilder();
+    stringBuilder.append("StatusMessage [");
+    stringBuilder.append("code=");
+    stringBuilder.append(getCode());
+    stringBuilder.append(", size=");
+    stringBuilder.append(getSize());
+    stringBuilder.append("]");
+    return stringBuilder.toString();
+  }
```

Comment:
I just spot the same missing `toString` doing some debugging, what do you think of this implementation?
```
  @Override
  public String toString() {
    return status().toString();
  }
```
and in the EthStatus inner class:
```
    @Override
    public String toString() {
      return "EthStatus{" +
          "protocolVersion=" + protocolVersion +
          ", networkId=" + networkId +
          ", totalDifficulty=" + totalDifficulty +
          ", bestHash=" + bestHash +
          ", genesisHash=" + genesisHash +
          ", forkId=" + forkId +
          '}';
    }
```

Reply:
thanks @fab-10 that makes sense, updated.